vendor/src/github.com/docker/engine-api/types/container/config.go

```diff
@@ -5,6 +5,20 @@ import (
 	"github.com/docker/go-connections/nat"
 )
 
+// HealthConfig holds configuration settings for the HEALTHCHECK feature.
+// For durations, a value of -1 means to inherit the default value.
+type HealthConfig struct {
+	Test        strslice.StrSlice // Test to perform to check container is healthy.
```

Comment:
That is no good. We don't want to multi-band this slice. There should be a separate type and data field.